packages/auth/package.json

```diff
@@ -11,7 +11,7 @@
     "build": "pnpm build:deps && tsc --build tsconfig.json",
     "clean": "rm -fr dist/",
     "fetch-schemas": "./scripts/get-op-schema.sh",
-    "test": "jest --passWithNoTests --maxWorkers=50%",
+    "test": "pnpm fetch-schemas && jest --passWithNoTests --maxWorkers=50%",
```

Comment:
In the spirit of https://github.com/interledger/rafiki/pull/684#discussion_r1009661953, I think we shouldn't cause a local dev to inadvertently nuke their local schemas.